slack_sdk/oauth/installation_store/amazon_s3/__init__.py

```diff
@@ -94,17 +120,30 @@ def save(self, installation: Installation):
             self.logger.debug(f"S3 put_object response: {response}")
 
     async def async_find_bot(
-        self, *, enterprise_id: Optional[str], team_id: Optional[str],
+        self,
+        *,
+        enterprise_id: Optional[str],
+        team_id: Optional[str],
+        is_enterprise_install: Optional[bool] = False,
```

Comment:
Just in case, I've aded a new flag to allow callers to specify enterprise installation even if the `team_id` arg is not None. As the arg has its default value, we have no breaking change by this.